### DISCUSSION TITLE
更新2.7版本后插件就在视频页面失效了，主页和动态仍然正常

### DISCUSSION THREAD

**Discussion by Zacccdp** [the1812/Bilibili-Evolved#4098] [提问 / Q&A] [answered]
在控制台搜索说是被禁用了，
![Screenshot 2023-03-26 165221](https://user-images.githubusercontent.com/74779279/227765392-02171f5b-dd8a-4d59-98e2-ac6d6dca1928.jpg)
看不懂代码，问下这是怎么回事

**Comment by Zacccdp** [comment, marked as answer]
重装了一遍也没用，有其他人遇到过这种情况的吗

**Reply by the1812** [reply]
和解除番剧区域限制同时使用时会禁用

**Reply by Zacccdp** [reply]
解决了，谢谢

**Comment by tkp206093** [comment]
<img width="1387" alt="螢幕擷取畫面 2023-03-26 183008" src="https://user-images.githubusercontent.com/29872453/227800912-791c9ea3-b60f-4717-a707-b4bc68bc714e.png">
